src/applications/hca/tests/e2e/utils/navigation.js

```diff
@@ -26,69 +25,6 @@ export const advanceToVaBenefits = (testData, props = {}) => {
   cy.selectRadio('root_vaCompensationType', compensation);
 };
 
-export const advanceToTERA = (testData, props = {}) => {
-  const {
-    birthdate = testData.veteranDateOfBirth,
-    entryDate = testData.lastEntryDate,
-    dischargeDate = testData.lastDischargeDate,
-  } = props;
-
-  startAsGuestUser();
-  fillIdentityForm({ ...testData, veteranDateOfBirth: birthdate });
-
-  goToNextPage('/veteran-information/birth-information');
-  goToNextPage('/veteran-information/maiden-name-information');
-  goToNextPage('/veteran-information/birth-sex');
-  cy.selectRadio('root_gender', 'M');
-
-  goToNextPage('/veteran-information/demographic-information');
-  goToNextPage('/veteran-information/veteran-address');
-  cy.fillAddress('root_veteranAddress', testData.veteranAddress);
-  cy.selectRadio('root_view:doesMailingMatchHomeAddress', 'Y');
-
-  goToNextPage('/veteran-information/contact-information');
-  goToNextPage('/va-benefits/basic-information');
-  cy.selectRadio('root_vaCompensationType', 'none');
-
-  goToNextPage('/va-benefits/pension-information');
-  cy.selectRadio('root_vaPensionType', 'No');
-
-  goToNextPage('/military-service/service-information');
-  cy.get('[name="root_lastServiceBranch"]').select(testData.lastServiceBranch);
-  cy.fillDate('root_lastEntryDate', entryDate);
-  cy.fillDate('root_lastDischargeDate', dischargeDate);
-  cy.get('[name="root_dischargeType"]').select(testData.dischargeType);
-
-  goToNextPage('/military-service/additional-information');
-  goToNextPage('/military-service/toxic-exposure');
-  cy.selectRadio('root_hasTeraResponse', 'Y');
-};
-
-export const advanceToHousehold = () => {
```

Comment:
Since we're starting in-progress, we don't need to advance to this section any longer.